keyboards/freyr/info.json

```
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "freyr", 
+    "url": "", 
+    "maintainer": "qmk", 
+    "width": 21.25, 
+    "height": 6.5, 
+    "layouts": {
+        "LAYOUT": {
```

This needs to match the layout used in the keyboard's header file, actually: 

```suggestion
        "LAYOUT_all": {
```

---

done